atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -224,13 +260,17 @@ def _get_replacement_pullspecs(self, pullspecs):
 
             self.log.info("Final pullspec: %s", replaced)
 
-            if replaced != original:
-                replacements[original] = replaced
+            replacements.append({
+                'original': original,
```

Comment:
convert it to str from ImageName

Reply:
I'd like to keep this as ImageName inside plugin (or workflow) for easier manipulation and potential future extensions. I.e converting only when needed.

Reply:
I'd rather have custom JSON encoder instead, so we don't have to explicitly retype to str.